packages/oc-template-es6/lib/render.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const es6Renderer = require('express-es6-template-engine');
```

Comment:
Note: For getting started I went with [this](https://github.com/dondido/express-es6-template-engine/blob/master/es6-renderer.js) , rationale:
- getting up and running quicker,
- very small code, well tested (the `express` specific code is very small, so we could think of a fork, in case we want to clean that down in a second step)
- allows for some nice thing like [precompiling](https://github.com/dondido/express-es6-template-engine/blob/master/test/es6-renderer.spec.js#L53) templates (as we do for handlebars), not adding that in in this first iteration but i think is something we might want to explore.